### DISCUSSION TITLE
Too few parameters obtained from Pipeline

### DISCUSSION THREAD

**Discussion by JiangYain** [open-webui/pipelines#235] [General]
While building Pipeline.py, I noticed that we were missing too many parameters passed in, such as UUID, front-end uploaded files, likes/dislikes, etc. Especially Chat_id, on the front end, I only generate such a uuid after sending a message when creating a new conversation, which makes me unable to create a new context when creating a new conversation. This is bad. I tried everything, but it didn't work.😥

**Comment by PlebeiusGaragicus** [comment]
I asked [something similar](https://github.com/open-webui/pipelines/issues/168)

Likes and dislikes should be handled somewhere else.  I think it's more for RLHF, anyways.  Maybe you're looking for the agent to react in real-time to thumbs - which is a cool idea.
